app/components/collections/edit/work_type_component.rb

```diff
@@ -19,6 +19,14 @@ def tooltip
         t('collections.edit.fields.work_subtypes.tooltip_html')
       end
 
+      def type_help_text
+        t('collections.edit.fields.work_type.help_text')
+      end
+
+      def subtype_help_text
+        t('collections.edit.fields.work_subtypes.help_text')
+      end
```

Comment:
It looks like `subtype_help_text` is defined in the work type component but all it does here is get injected into the work subtypes component. How about moving this method directly into the work subtypes component where it's used?